calyx-py/calyx/builder.py

```diff
@@ -943,6 +943,60 @@ def neq_store_in_reg(
         cell = self.neq(width, cellname, signed)
         return self.op_store_in_reg(cell, left, right, cell.name, 1, ans_reg)
 
+    def pipe_store_in_reg(
```

Comment:
Nice! A little thought, in case you wanted to have a little more "fun" in Python: this code is really quite similar to `op_store_in_reg`, and I now wonder if you want to merge this functionality into that method.

At a first level, you could literally just have an optional parameter, `is_comb`, set to False by default. If `is_comb` is true, your group's body would change a little. See the method `mem_load_d1` in the same file for an example of how to add wires to a group based on some Python-level check! Let's have you try this out before we merge.

The next level, for double fun, is that I bet you can actually infer `is_comb` based on what cell you were passed. We do not currently have a method `is_comb()` defined on cells, but it should not be all that hard to define. You'd have to look at the cell's `operation` field, which is a string like "add" or "mult_pipe". Let me know if pursuing this goal, and I'll jump in with you.

Reply:
I think that's a great idea! I can get to work on that.
Was this the side quest you mentioned above?

Reply:
Yes I am requesting the first level for sure, and the second level if you want to pursue it. Can keep doing it in this branch+PR!

Reply:
I think it should be good to go now!
For the time being, I'm keeping it such that `is_comb` just checks the operation against a list of names. Can optimize it further later perhaps, but it works well for now.